src/features/groups/components/InviteMemberDialog/InviteMemberDialogContent.tsx

```diff
@@ -7,9 +7,10 @@ import {
   DialogTitle,
   Icon,
 } from '@/components/ui';
+import { generateInvitationLinkHash } from '@/lib/api/group/client/groupApiClient.client';
 
 import { useEffect, useState } from 'react';
-
+const BACKEND_API_DOMAIN = process.env.NEXT_PUBLIC_BACKEND_API_DOMAIN || '';
```

Comment:
Please add a line break before each function declaration. If it has a comment block, then the line break should be right before the comment.